src/org/labkey/test/components/glassLibrary/components/BaseReactSelect.java

```diff
@@ -79,6 +79,11 @@ public boolean isClearable()
         return hasClass("is-clearable");
     }
 
+    public boolean isDisabled()
+    {
+        return hasClass("is-disabled");
+    }
```

Comment:
Consider inverting this to be consistent with the `WebElement` interface.
```suggestion
    public boolean isEnabled()
    {
        return !hasClass("is-disabled");
    }
```

Reply:
Haha, it turns out I'd already implemented 'isEnabled' on it